src/modules/75-cf/pages/feature-flags/__tests__/FeatureFlagsPage.test.tsx

```diff
@@ -139,6 +139,14 @@ describe('FeatureFlagsPage', () => {
     })
   })
 
+  test('It should have a link to FF documentation', async () => {
+    renderComponent()
+
+    expect(screen.getByRole('link', { name: 'cf.shared.readDocumentation' }).getAttribute('href')).toBe(
+      'https://developer.harness.io/docs/feature-flags/ff-onboarding/cf-feature-flag-overview'
+    )
```

Comment:
Instead of using `getAttribute` and `toBe`, try `toHaveAttribute`
```suggestion
    expect(screen.getByRole('link', { name: 'cf.shared.readDocumentation' })).toHaveAttribute(
      'href',
      'https://developer.harness.io/docs/feature-flags/ff-onboarding/cf-feature-flag-overview'
    )
```